specs/unified_kernel_image.md

```diff
@@ -40,22 +40,45 @@ that glues the listed components together, generating one PE binary that then ca
 UKIs consist of the following resources:
 
 * An UEFI boot stub that forms the initial program.
-  It contains various sections normally required for a program,
+  It contains various PE sections normally required for a program,
   including `.text`, `.reloc`, `.data`, and others.
-* The Linux kernel in the `.linux` section.
-* The initrd that the kernel shall unpack and invoke, in the `.initrd` section.
-* Optionally, the kernel command line in the `.cmdline` section. If this is absent, the loader implementation may allow local overrides instead.
+* The Linux kernel in the `.linux` PE section.
 * Optionally, information describing the OS this kernel is intended for, in the `.osrel` section. The contents of this section are derived from `/etc/os-release` of the target OS. They can be useful for presentation of the UKI in the boot loader menu, and ordering it against other entries using the included version information.
-* Optionally, information describing kernel release information (i.e. `uname -r` output) in the `.uname` section. This is also useful for presentation of the UKI in the boot loader menu, and ordering it against other entries.
+* Optionally, the kernel command line in the `.cmdline` section. If this is absent, the loader implementation may allow local overrides instead.
+* The initrd that the kernel shall unpack and invoke, in the `.initrd` section.
```

Comment:
"Optionally, "?

Reply:
Or maybe let's do this change as as a separate commit. I'll rebase #83 and do this change there.